src/main/java/uk/gov/hmcts/reform/cwrdapi/controllers/CaseWorkerRefController.java

```diff
@@ -132,4 +139,55 @@
                 .status(idamRolesMappingResponse.getStatusCode())
                 .body(idamRolesMappingResponse);
     }
+
+    @ApiOperation(
+            value = "This API gets the User details from Caseworker Profile",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
+    )
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "Successfully fetched the Caseworker profile(s)",
+                    response = CaseWorkerProfile.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = BAD_REQUEST
+            ),
+            @ApiResponse(
+                    code = 401,
+                    message = UNAUTHORIZED_ERROR
+            ),
+            @ApiResponse(
+                    code = 403,
+                    message = FORBIDDEN_ERROR
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = NO_DATA_FOUND
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = INTERNAL_SERVER_ERROR
+            )
+    })
+    @PostMapping(
+            path = "/users/fetchUsersById/",
+            consumes = APPLICATION_JSON_VALUE,
+            produces = APPLICATION_JSON_VALUE
+    )
+    @Secured("cwd-admin")
```

Comment:
Should not this be `org-role-cwd-sytem-user`

Reply:
Access Management team is yet to create this role. Chasing with the IDAM team.

Reply:
So if merge this then we will be in need of a ticket to change this once AM team creates the rol - FYI @pksaxena01

Reply:
I have put a comment on RDCC-2059 for this pending IDAM role.

Reply:
Thanks :)